Makefile

```diff
@@ -32,11 +32,10 @@ install-safety:
 
 check-dependencies: install-safety ## Scan dependencies for security vulnerabilities
 	# 12 Dec 2023: 51668 is fixed with >= 2.0.0b1 of SQLAlchemy. Ongoing refactor to upgrade.
-	# 6 June 2024: 70624 will be resolved with ticket #1794
+	# 6 June 2024: 70624 will be resolved with ticket #1794 - flask-cors
```

Comment:
flask-cors was updated in this PR to 4.0.1, which clears the vulnerability. l35 can be deleted.